zoo/services/models.py

```diff
@@ -79,6 +79,12 @@ class Meta:
         default=list,
         help_text="Used for filtering and defining presets of checks",
     )
+    exclude_security_checks = models.CharField(
```

Comment:
We are going to use it for our security checks for now, yes, but this field should be called something more generic, like `exclusions` or something like that.